corelib/crayfish_renderer.cpp

```diff
@@ -107,6 +107,22 @@ void Renderer::draw()
     drawVectorData(mOutputVector);
 }
 
+void Renderer::drawElementMeshPolyline(QPainter& p, const Element& elem) {
+    int nPoints = elem.nodeCount();
+    QPoint pts[nPoints + 1];
```

Comment:
I think this is just some gcc extension to define dynamic array like this. In Qt there is also QPolygon class which suitable here...